datasets/au/dfat_sanctions/au_dfat_sanctions.yml

```
@@ -39,8 +39,7 @@ data:
   # url: "https://data.opensanctions.org/datasets/20231015/au_dfat_sanctions/source.xlsx"
   format: XLSX
 dates:
-  formats: ["%Y-%m-%d", "%d/%m/%Y", "%d %b. %Y", "%d %b.%Y", "%d %b %Y", "%d %B %Y", "%b. %Y", "%d %B. %Y", "%Y", "%m/%Y"]
-
+  formats: ["%Y-%m-%d", "%d/%m/%Y", "%d %b. %Y", "%d %b.%Y", "%d %b %Y", "%d %B %Y", "%b. %Y", "%d %B. %Y", "%Y", "%m/%Y", "%B %Y", "%d-%b-%Y", "%b %Y", "%d. %b %Y", "c. %Y", "%B, %Y"]
```

at this point it can become
```suggestion
  formats:
    - "%Y-%m-%d"
    - "%d/%m/%Y"
    - ...
```